### PR TITLE
PLATUI-2883 fix up missing/incorrect Visual Regression Tests and rebase with 0% misMatchThreshold

### PR DESCRIPTION
**Pull request by ellamdav**

### CODE REVIEW

File: js/test/backstop/backstopConfig.js [hunk 1]
```diff
@@ -47,15 +47,15 @@ module.exports = ({ host, port }) => ({
...
-      onReadyScript: 'tabToCookiesLink.js',
+      onReadyScript: 'tabToAcceptAdditional.js',
       readySelector: '.cbanner-govuk-cookie-banner',
     },
     {
-      label: 'Tracking Consent - Cookie Banner - Tab to accept additional',
+      label: 'Tracking Consent - Cookie Banner - Tab to reject additional',
       url: `http://${host}:${port}/`,
-      onReadyScript: 'tabToAcceptAdditional.js',
+      onReadyScript: 'tabToRejectAdditional.js',
```
Comment (ellamdav) [inline js/test/backstop/backstopConfig.js:58]: added missing tests for this scenario

File: js/test/backstop/backstopConfig.js [hunk 2]
```diff
@@ -139,21 +139,27 @@ module.exports = ({ host, port }) => ({
       hoverSelector: '.cbanner-govuk-button[value=reject]',
     },
     {
-      label: 'Tracking Consent - Cookie Banner - Classic Services - Tab to Link',
+      label: 'Tracking Consent - Cookie Banner - Classic Services - Tab to Skip Link',
```
Comment (ellamdav) [inline js/test/backstop/backstopConfig.js:142]: renamed for consistency

File: js/test/backstop/backstopConfig.js [hunk 3]
```diff
@@ -139,21 +139,27 @@ module.exports = ({ host, port }) => ({
...
+    {
+      label: 'Tracking Consent - Cookie Banner - Classic Services - Tab to reject additional',
+      url: `http://${host}:${port}/classic-services.html`,
+      onReadyScript: 'tabToRejectAdditional.js',
       readySelector: '.cbanner-govuk-cookie-banner',
     },
     {
       label: 'Tracking Consent - Cookie Banner - Classic Services - Tab to Cookie Settings',
       url: `http://${host}:${port}/classic-services.html`,
-      onReadyScript: 'tabToAcceptAdditional.js',
```
Comment (ellamdav) [inline js/test/backstop/backstopConfig.js:156]: fixed these scripts which were somehow transposed

File: js/test/backstop/backstopConfig.js [hunk 4]
```diff
@@ -173,7 +179,8 @@ module.exports = ({ host, port }) => ({
       '--no-sandbox',
     ],
   },
-  asyncCaptureLimit: 5,
+  misMatchThreshold: 0,
```
Comment (ellamdav) [inline js/test/backstop/backstopConfig.js:182]: reduce to 0 as previous `View cookie[ preference]s` change hadn't been picked up

**Review by TimothyFothergill — APPROVED** [review]
applause

**Review by oscarduignan — APPROVED** [review]

**Pull request merged**